test/api/requests/build.test.js

```diff
@@ -179,54 +183,19 @@ describe('Build API', () => {
           })
           .then((response) => {
             validateAgainstJSONSchema('POST', '/build', 200, response.body);
-            return Build.findOne({
+            return Build.findAll({
```

Comment:
hmmmm, I *think* `findOne` is the same things a `findAll` except it just returns only the first record...

Reply:
changed